sig-cluster-lifecycle/annual-report-2022.md

```diff
@@ -4,64 +4,97 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
-
+   - We migrated the default image repository in kubeadm and some of our other k8s deployment
+   tools from k8s.gcr.io to registry.k8s.io.
+   - kubeadm initiated work on migrating its etcd bootstrapping to use ["learner mode"](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cluster-lifecycle/kubeadm/3614-etcd-learner-mode)
+   - We finilized migrating subprojects away from the legacy ["master" label/taint](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cluster-lifecycle/kubeadm/2067-rename-master-label-taint)
+   - We completed the setup of the ["kubernetes-sigs/cluster-api-addon-provider-helm"](https://github.com/kubernetes-sigs/cluster-api-addon-provider-helm/blob/main/OWNERS) subproject providing an option for Cluster API users to manage addons, and most specifically CPI being moved out of tree.
+   - We created the ["logical-cluster"](https://github.com/kubernetes-sigs/logical-cluster/blob/main/OWNERS) subproject to help in cases of CAPI cluster fleets.
+   - We worked tighly with test-infra supporting the migration to registry.k8s.io as well as reviewing test footprint of Cluster API.
+   - We started a Cluster API release team modeled from the Kubernetes release team, and this allowed the introduction of a quarterly release cadence for the entire 2023.
```

Comment:
Is there a doc pointing to the regular cadence? We can link it here for public visibility.